packages/sources/xbto/tsconfig.json

```diff
@@ -3,10 +3,11 @@
   "compilerOptions": {
     "outDir": "dist",
     "rootDir": "src",
+    "types": ["mocha"],
     "typeRoots": ["../../../node_modules/@types", "../../core/types", "./types"],
     "composite": true
   },
   "include": ["src/**/*"],
   "exclude": ["dist", "**/*.spec.ts", "**/*.test.ts"],
-  "references": [{ "path": "../../core/external-adapter" }, { "path": "../../core/bootstrap" }]
+  "references": [{ "path": "../../core/bootstrap" }]
```

Comment:
Why did we add these references in? Is that to support topologically building dependencies?

We were using `wsrun` for that with Yarn 1, but now with Yarn 2 there is a [foreach  -t](https://yarnpkg.com/cli/workspaces/foreach) option that should do just that.

Reply:
[References paths](https://www.typescriptlang.org/docs/handbook/project-references.html) are supposed to speed up build times and improve intellisense performance by referencing already compiled code when building dependent projects.

Is that what the `-t` flag on `foreach` is doing too?